createuserpkg

```diff
@@ -119,7 +135,10 @@ def main():
         user_data['shell'] = options.shell
     if options.hidden:
         user_data['IsHidden'] = u'YES'
-
+    if options.picture:
+        from locallibs import imageutil
```

Comment:
imports really belong at the top of a Python file unless you have a really good reason to do them conditionally.

Reply:
I thought i only import it if there is a image as argument, but i will put it on top